fe/fe-core/src/main/java/com/starrocks/connector/hive/HiveMetastoreApiConverter.java

```diff
@@ -193,6 +199,9 @@ public static List<Column> toFullSchemasForHudiTable(Schema hudiSchema) {
         List<Schema.Field> allHudiColumns = hudiSchema.getFields();
         List<Column> fullSchema = Lists.newArrayList();
         for (Schema.Field fieldSchema : allHudiColumns) {
+            if (HUDI_INTERNAL_FIELD_NAMES.contains(fieldSchema.name())) {
```

Comment:
These can't be hide by default, you can add a property in create catalog statement to let user decide whether to hide it or not.

Reply:
Agree.
Better to keep it and these fields are valuable when debugging

Reply:
agree